### PR TITLE
Convert specs to RSpec 3.5.4 syntax with Transpec

### PR DESCRIPTION
**Pull request by ferdinandrosario**
This conversion is done by Transpec 3.3.0 with the following command:
    transpec -f

* 116 conversions
    from: obj.should
      to: expect(obj).to

* 98 conversions
    from: it { should ... }
      to: it { is_expected.to ... }

* 92 conversions
    from: obj.should_receive(:message)
      to: expect(obj).to receive(:message)

* 59 conversions
    from: obj.stub(:message)
      to: allow(obj).to receive(:message)

* 53 conversions
    from: == expected
      to: eq(expected)

* 36 conversions
    from: it { should_not ... }
      to: it { is_expected.not_to ... }

* 12 conversions
    from: obj.should_not
      to: expect(obj).not_to

* 2 conversions
    from: Klass.any_instance.should_not_receive(:message)
      to: expect_any_instance_of(Klass).not_to receive(:message)

* 2 conversions
    from: obj.should_not_receive(:message)
      to: expect(obj).not_to receive(:message)

* 2 conversions
    from: obj.stub(:message => value)
      to: allow(obj).to receive_messages(:message => value)

* 2 conversions
    from: obj.stub_chain(:message1, :message2)
      to: allow(obj).to receive_message_chain(:message1, :message2)

* 1 conversion
    from: Klass.any_instance.should_receive(:message)
      to: expect_any_instance_of(Klass).to receive(:message)

* 1 conversion
    from: lambda { }.should
      to: expect { }.to

For more details: https://github.com/yujinakayama/transpec#supported-conversions

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (chrisarcand) [inline .idea/vcs.xml]: I don't think this file is needed. Can you remove?

**Comment by ferdinandrosario** [comment]
sure I will @chrisarcand. let me do it..

**Comment by chrisarcand** [comment]
Thanks!

**Comment by ferdinandrosario** [comment]
:+1:  Thanks @chrisarcand

**Pull request merged**